### PR TITLE
[Not-for-review] Sample: deviceupdate

### PR DESCRIPTION
**Pull request by chunyu3**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

- A sample PR to show our approach to handle LRO method which has no return value
-  re-gen all the RPs in the azure-sdk-for-net via the new autorest.csharp

### CODE REVIEW

File: sdk/cognitivelanguage/Azure.AI.Language.Conversations/src/Generated/ConversationAnalysisClient.cs [hunk 1]
```diff
@@ -350,7 +350,7 @@ public virtual Response JobStatus(Guid jobId, bool? showStats = null, RequestCon
         /// </code>
         /// 
         /// </remarks>
-        public virtual async Task<Operation<BinaryData>> SubmitJobAsync(WaitUntil waitUntil, RequestContent content, RequestContext context = null)
+        public virtual async Task<Operation> SubmitJobAsync(WaitUntil waitUntil, RequestContent content, RequestContext context = null)
```
Comment (annelo-msft) [inline sdk/cognitivelanguage/Azure.AI.Language.Conversations/src/Generated/ConversationAnalysisClient.cs:353]: @heaths, could you confirm that there is no return-type from the `SubmitJob` LRO?

File: sdk/cognitivelanguage/Azure.AI.Language.QuestionAnswering/src/Generated/QuestionAnsweringProjectsClient.cs [hunk 2]
```diff
@@ -2107,7 +2107,7 @@ IEnumerable<Page<BinaryData>> CreateEnumerable(string nextLink, int? pageSizeHin
         /// </code>
         /// 
         /// </remarks>
-        public virtual async Task<Operation<BinaryData>> DeleteProjectAsync(WaitUntil waitUntil, string projectName, RequestContext context = null)
+        public virtual async Task<Operation> DeleteProjectAsync(WaitUntil waitUntil, string projectName, RequestContext context = null)
```
Comment (annelo-msft) [inline sdk/cognitivelanguage/Azure.AI.Language.QuestionAnswering/src/Generated/QuestionAnsweringProjectsClient.cs:2110]: @heaths, similarly, could you confirm that `DeleteProject` doesn't return a value?  Thanks!

File: sdk/deviceupdate/Azure.IoT.DeviceUpdate/src/Generated/DeviceManagementClient.cs [hunk 3]
```diff
@@ -3679,7 +3679,7 @@ IEnumerable<Page<BinaryData>> CreateEnumerable(string nextLink, int? pageSizeHin
         /// </code>
         /// 
         /// </remarks>
-        public virtual async Task<Operation<BinaryData>> ImportDevicesAsync(WaitUntil waitUntil, string action, RequestContent content, RequestContext context = null)
+        public virtual async Task<Operation> ImportDevicesAsync(WaitUntil waitUntil, string action, RequestContent content, RequestContext context = null)
```
Comment (annelo-msft) [inline sdk/deviceupdate/Azure.IoT.DeviceUpdate/src/Generated/DeviceManagementClient.cs:3682]: @mikekistler, can you confirm that there is no value returned from the `ImportDevices` LRO?

File: sdk/cognitivelanguage/Azure.AI.Language.Conversations/src/Generated/ConversationAnalysisClient.cs (hunk 1 above)
Comment (chunyu3) [reply sdk/cognitivelanguage/Azure.AI.Language.Conversations/src/Generated/ConversationAnalysisClient.cs:353]: yes, I confirmed that SubmitJob will not return an value.

Comment (heaths) [reply sdk/cognitivelanguage/Azure.AI.Language.Conversations/src/Generated/ConversationAnalysisClient.cs:353]: That is correct: it does not return data.

File: sdk/cognitivelanguage/Azure.AI.Language.QuestionAnswering/src/Generated/QuestionAnsweringProjectsClient.cs (hunk 2 above)
Comment (annelo-msft) [reply sdk/cognitivelanguage/Azure.AI.Language.QuestionAnswering/src/Generated/QuestionAnsweringProjectsClient.cs:2110]: Thanks, Heath!

File: sdk/deviceupdate/Azure.IoT.DeviceUpdate/src/Generated/DeviceUpdateClient.cs [hunk 4]
```diff
@@ -1535,7 +1535,7 @@ public virtual async Task<Operation<BinaryData>> DeleteUpdateAsync(WaitUntil wai
         /// </code>
         /// 
         /// </remarks>
-        public virtual Operation<BinaryData> DeleteUpdate(WaitUntil waitUntil, string provider, string name, string version, RequestContext context = null)
+        public virtual Operation DeleteUpdate(WaitUntil waitUntil, string provider, string name, string version, RequestContext context = null)
```
Comment (annelo-msft) [inline sdk/deviceupdate/Azure.IoT.DeviceUpdate/src/Generated/DeviceUpdateClient.cs:1538]: @mikekistler, can you confirm that there is no value returned from the `DeleteUpdate` LRO?

File: sdk/deviceupdate/Azure.IoT.DeviceUpdate/src/Generated/DeviceManagementClient.cs (hunk 3 above)
Comment (mikekistler) [reply sdk/deviceupdate/Azure.IoT.DeviceUpdate/src/Generated/DeviceManagementClient.cs:3682]: I cannot confirm what is returned from this operation. I had hoped to create a test environment to determine this empirically but it proved too complicated.

File: sdk/deviceupdate/Azure.IoT.DeviceUpdate/src/Generated/DeviceUpdateClient.cs (hunk 4 above)
Comment (mikekistler) [reply sdk/deviceupdate/Azure.IoT.DeviceUpdate/src/Generated/DeviceUpdateClient.cs:1538]: I cannot confirm what is returned from this operation.  I hoped to determine this empirically by setting up a test environment but it proved too complicated.

Comment (annelo-msft) [reply sdk/deviceupdate/Azure.IoT.DeviceUpdate/src/Generated/DeviceUpdateClient.cs:1538]: @dpokluda, could you confirm that there is no value returned from the DeleteUpdate LRO?

File: sdk/deviceupdate/Azure.IoT.DeviceUpdate/src/Generated/DeviceManagementClient.cs (hunk 3 above)
Comment (annelo-msft) [reply sdk/deviceupdate/Azure.IoT.DeviceUpdate/src/Generated/DeviceManagementClient.cs:3682]: @dpokluda, could you confirm that there is no value returned from the ImportDevices LRO?

**Pull request closed without merging**